tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = py27
+
+[testenv]
+basepython = python
+
+deps =
+    coverage
+    flake8
+    pytest
+    mock
+    requests
+    treeherder-client
+
+commands =
+    flake8 triggerbot test buildapistats
+    coverage run --source=triggerbot -m py.test test
```

Comment:
Ok, that looks reasonable, it must be automatically detecting the tests. Thanks.